### PR TITLE
Remove the leading slash from the directory label

### PR DESCRIPTION
**Pull request by amarakon**
See #52.

### CODE REVIEW

**Comment by hrsh7th** [comment]
I prefer the slashes, what should I do?

**Comment by amarakon** [comment]
> I prefer the slashes, what should I do?

Maybe it should be enabled by default but add an option to disable it.

File: README.md [hunk 1]
```diff
@@ -37,6 +37,12 @@ _Default:_ `false`
 
 Specify if completed directory names should include a trailing slash. Enabling this option makes this source behave like Vim's built-in path completion.
 
+### visual_slash (type: boolean)
```
Comment (hrsh7th) [inline README.md:40]: label_trailing_slash is better name.
and default value seems to,wrong.

Comment (amarakon) [reply README.md:40]: Sure, change it to `label_trailing_slash` if you wish. I changed the default value to true which shows it, but in my config I will set it to false. Set the default to false if you think that's a better default.

Comment (hrsh7th) [reply README.md:40]: Yes. please change :)

Comment (amarakon) [reply README.md:40]: Also, should the default be `true` or `false`?

Comment (hrsh7th) [reply README.md:40]: label_trailing_slash should be true by defailt

Comment (amarakon) [reply README.md]: Okay, the latest commit should be good.

File: README.md [hunk 2]
```diff
@@ -37,6 +37,12 @@ _Default:_ `false`
 
 Specify if completed directory names should include a trailing slash. Enabling this option makes this source behave like Vim's built-in path completion.
 
+### label_trailing_slash (type: boolean)
+
+_Default:_ `false`
```
Comment (hrsh7th) [inline README.md:42]: default  is true.

**Comment by hrsh7th** [comment]
LGTM. Thank you.

**Pull request merged**